inherited_resources.gemspec

```diff
@@ -14,7 +15,9 @@ Gem::Specification.new do |s|
 
   s.rubyforge_project = "inherited_resources"
 
-  s.files         = Dir["app/**/*", "lib/**/*", "README.md", "MIT-LICENSE"]
+  s.files         = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
+  s.bindir        = "exe"
+  s.executables   = s.files.grep(%r{^exe/}) { |f| File.basename(f) }
```

Comment:
Even if these are bundler's default, I don't think we need the prior two lines because this gem has no CLI.

Reply:
Sounds good to me! I will remove them. 👍🏻